package.json

```diff
@@ -3024,6 +3118,7 @@
                     "hidden",
                     "inherit"
                   ],
+                  "default": "hidden",
```

Comment:
What effects does making this hidden by default cause?

Reply:
Looks like this is specific to the workflow preset, so this is a slightly different intent than the rest of this PR. As long as we're okay with this change, I'm okay with it.